languages/fr-FR.js

```diff
@@ -105,15 +105,16 @@ module.exports = class extends Language {
 				'Une grande partie du code est modularisée, ce qui permet aux développeurs de modifier Klasa pour répondre à leurs besoins.',
 				'',
 				'Les fonctionnalités de Klasa comprennent :',
-				'• Temps de chargement rapide avec le support de l\'ES7 (Async/Await)',
-				'• Paramètres par serveur, qui peuvent être étendus avec votre propre code',
-				'• Système de commandes personnalisable avec l\'analyse automatique de l\'usage ainsi qu\'un téléchargement et rechargement de modules faciles à utiliser',
-				'• "Moniteurs" qui peuvent observer et agir sur les messages, comme un évenement message normal (Filtre à Injures, Spam Protection, etc)',
-				'• "Inhibiteurs" qui peuvent empêcher l\'exécution d\'une commande en fonction de paramètres (Permissions, Blacklists, etc)',
-				'• "Fournisseurs" qui vous permettent de vous connecter à une base de données externe de votre choix.',
-				'• "Finaliseurs" qui s\'exécutent après une commande réussie.',
-				'• "Extendables", code qui agit passivement. Ils ajoutent des propriétés et des méthodes aux classes existantes de Discord.js.',
-				'• "Langages", qui vous permettent de localiser votre bot.',
+				'• 🐇💨 Temps de chargement rapide avec le support de l\'ES2017 (`async`/`await`)',
+				'• 🎚🎛 Paramètres par serveur, qui peuvent être étendus avec votre propre code',
+				'• 💬 Système de commandes personnalisable avec l\'analyse automatique de l\'usage ainsi qu\'un téléchargement et rechargement de modules faciles à utiliser',
+				'• 👀 "Moniteurs" qui peuvent observer et agir sur les messages, comme un évenement message normal (Filtre à Injures, Spam Protection, etc)',
+				'• ⛔ "Inhibiteurs" qui peuvent empêcher l\'exécution d\'une commande en fonction de paramètres (Permissions, Blacklists, etc)',
+				'• 🗄 "Fournisseurs" qui vous permettent de vous connecter à une base de données externe de votre choix.',
+				'• ✅ "Finaliseurs" qui s\'exécutent après une commande réussie.',
+				'• ➕ "Extendables", code qui agit passivement. Ils ajoutent des propriétés et des méthodes aux classes existantes de Discord.js.',
+				'• 🌐 "Langages", qui vous permettent de localiser votre bot.',
+				'• ⏲ <"Tasks" needs translated>',
```

Comment:
I would say : `"Tâches", qui peuvent être planifiées pour s'exécuter dans le futur, potentiellement de manière récurrente`  
I'm not really satisfied with the end, but this should do it anyway.